tests/components/airq/test_config_flow.py

```diff
@@ -94,3 +93,21 @@ async def test_form_invalid_input(hass: HomeAssistant) -> None:
 
     assert result2["type"] == FlowResultType.FORM
     assert result2["errors"] == {"base": "invalid_input"}
+
+
+async def test_duplicate_error(hass: HomeAssistant) -> None:
+    """Test that errors are shown when duplicates are added."""
```

Comment:
Please remove this and keep it inside #90192

Reply:
This slipped through by my oversight, my apologies